src/concrete/ml/sklearn/xgb.py

```diff
@@ -331,6 +331,14 @@ def fit(self, X, y, *args, **kwargs) -> Any:
         super().fit(X, y, *args, **kwargs)
         return self
 
+    def post_processing(self, y_preds: numpy.ndarray) -> numpy.ndarray:
+        y_preds = super().post_processing(y_preds)
+
+        # Hummingbird Gemm for XGBoostRegressor adds a + 0.5 at the end of the graph.
+        # We need to add it back here since the graph is cut before this add node.
+        y_preds += 0.5
```

Comment:
are you sure it's always 0.5 ? Can you maybe add an assert in the onnx parser ?

Reply:
It's a constant being added called `base_prediction`. We can add an assert yes.

Reply:
Done.